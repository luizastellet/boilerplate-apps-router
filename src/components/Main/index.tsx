import * as S from './styles'

function Main() {
  return (
    <S.Wrapper>
      <h1>main</h1>
    </S.Wrapper>
  )
}

export default Main
