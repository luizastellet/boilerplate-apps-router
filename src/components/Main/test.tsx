import { render, screen } from "@testing-library/react"
import Main from "."

describe(('Main'), () => {
  it('should be able to render an heading', () => {
    // renderiza o componente
    // busca o elemento e verifica se ele existe
    const { container } = render(<Main />)
    expect(screen.getByRole('heading', { name: /main/i })).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
