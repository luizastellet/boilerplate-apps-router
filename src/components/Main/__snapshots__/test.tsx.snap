// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main should be able to render an heading 1`] = `
<main>
  <h1>
    main
  </h1>
</main>
`;
